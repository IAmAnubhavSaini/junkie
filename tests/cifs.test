#!/bin/sh

NAME="CIFS"
PCAPS="pcap/smb/smb.pcap"
filter() {
    grep -E '^(Netbios|CIFS)'
}
OUTPUT="cifs.out"

. "$srcdir/dump_test"
