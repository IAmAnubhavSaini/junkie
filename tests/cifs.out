-- pcap/smb/smb.pcap --
Netbios: head_len=4, payload=70
CIFS: head_len=32, payload=38, command=0x32, status=0x00000000
Netbios: head_len=4, payload=35
CIFS: head_len=32, payload=3, command=0x32, status=0x007c0001
Netbios: head_len=4, payload=84
CIFS: head_len=32, payload=52, command=0x06, status=0x00000000
Netbios: head_len=4, payload=35
CIFS: head_len=32, payload=3, command=0x06, status=0xc0000034
